index.html

```diff
@@ -0,0 +1,56 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Zuhra Hashimi</title>
+</head>
+<body>
+    <header>
+        <nav class="topnav">
+            <a href="#" class="logo">Zuhra Hashimi</a>
+            <div class="page_header">
+                <img class="img" src="images/Icon - Menu.png" alt="">
+            </div>
+        </nav>
+    </header>
+    <section>
+        <section class="main-content">
+            <div class="container">
+                <h1 class="container headline">Hey There.<br>I'm Zuhra</h1>
+                <h2 class="container job-title">I am a Software developer </h2>
+                <p class="container job-description">I can help you build a product,
+                    feature or
+                    website Look through some of my work
+                    and experience! If you like what you see
+                    and have a project you need coded,
+                    do not hestiate to contact me.</p>
+            </div>
+    </section>
+    <div class="links">
+        <ul class="social_media">
+            <li>
+                <a href="https://github.com/ZuhRa-HashiMi" id="git" target="blank"><img src="images/github.svg" alt="github"></a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/zuhra-hashimi-601966214/" id="linkedin" target="blank"><img src="images/Linkedin.svg" alt="linkedin"></a>
+            </li>
+         
+            <li>
+                <a href="https://twitter.com/ZuhraHashimi4" id="twitter" target="blank"><img src="images/twitter.svg" alt="twitter"></a>
+            </li>
+            <li>
+                <a href="http://medium.com" id="medium" target="blank"><img src="images/Linkedin.svg" alt="linkedin"></a>
+            </li>
+            <li>
+                <a href="http://angellist.com" id="angel-list" target="blank"><img src="images/angellist.svg" alt="angel-list"></a>
+            </li>
+     
+        </ul>
+    </div>  
```

Comment:
[OPTIONAL] It'd be better to move these links into the `main-content` section, this way, you will only make one section for the `headline` and it'd also align the icons with the rest of the design as you can see below they are not perfectly aligned right now 🤗

![image](https://user-images.githubusercontent.com/73714615/191233176-6fa35778-b013-4979-a678-c2df6fa17191.png)